ui/cli-lib/src/Ariadne/UI/Cli.hs

```diff
@@ -76,13 +104,11 @@ isHelpCommand t = strip t `elem` ["help"]
 outputTextLn :: Text -> Repl ()
 outputTextLn = Haskeline.outputStrLn . toString
 
-getInputTextLine :: Text -> Repl (Maybe Text)
-getInputTextLine = (fmap.fmap) toText . Haskeline.getInputLine . toString
+getInputTextLine :: Repl (Maybe Text)
+getInputTextLine = getUsingText Haskeline.getInputLine "knit> "
 
-data LoopDecision = Continue | Stop
+getTextPassword :: Repl (Maybe Text)
+getTextPassword = getUsingText (Haskeline.getPassword (Just '*')) "password> "
```

Comment:
Please make the prompt `Password: `, like it's usually in shells.
Also, I think we should disable asterisks in the same vein.